addons/binding/org.openhab.binding.oceanic/ESH-INF/thing/softener.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="oceanic" xmlns:xsi="http://www.w3.org/2001/XLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="softener">
+		<label>Oceanic Water Softener</label>
+		<description>Oceanic Water Softener (www.oceanic.be)</description>
+
+		<properties>
+			<property name="serial"></property>
```

Comment:
None. The existence of certain properties does not need to be declared. Your handler can just set whatever it wants. No need to mention anything in the XML.